src/components/progress-indicators/WalletSyncCircle.tsx

```diff
@@ -60,11 +57,13 @@ export const WalletSyncCircle = (props: Props) => {
   const viewBox = `${-strokeWidth * 2} ${-strokeWidth * 2} ${svgSize} ${svgSize}`
 
   // Animated Params
-  const animatedProps = useAnimatedProps(() => ({
-    strokeDashoffset: circumference * (1 - syncRatio.value),
-    opacity: withTiming(syncRatio.value === 1 ? 0 : 1, { duration: 500 }),
-    stroke: withTiming(isDone.value ? theme.walletProgressIconDone : theme.walletProgressIconFill)
-  }))
+  const animatedProps = useAnimatedProps(() => {
+    return {
```

Comment:
Why add the `return` statement? The `=> ({})` syntax is nice because it says "there is no logic in here," but I can also see the `return` syntax being nice for letting you easily add debug stuff to the lines above.